lib/components/ConflictPicker.vue

```diff
@@ -2,17 +2,18 @@
 	<NcDialog class="conflict-picker"
 		data-cy-conflict-picker
 		:close-on-click-outside="false"
-		:can-close="false"
+		:can-close="true"
 		:show="opened"
 		:name="name"
 		size="large"
-		@close="onCancel">
+		@closing="onCancel">
```

Comment:
This one is emitted before the dialog is really closed, if you need it to be emitted when it is not visible anymore use `update:open` and check its parameter to be `false`.

Reply:
I saw this, but I think it doesn't really matter, we are cancelling everything.
So if it's closing, we can start the cleanup :broom: